frontend/src/routes/+page.svelte

```diff
@@ -1,2 +1,174 @@
+        </figcaption>
+      </figure>
+    </div>
+    <div class="card bordered">
+      <figure>
+        <figcaption class="p-4 card-body">
+          <h2 class="card-title">Time-Bound</h2>
+          <p id="time-bound" class="text-gray-600" contenteditable="true">
+            Information from API...
+          </p>
+        </figcaption>
+      </figure>
+    </div>
+  </div>
+</div>
+
+<div class="mt-4 flex flex-col items-center">
+  <span class="text-gray-700 font-bold text-lg">Days of the week</span>
+  <div class="mt-2 space-y-2 w-1/2">
+    <div class="rounded shadow p-2">
+      <label class="inline-flex items-center">
+        <input
+          type="checkbox"
+          class="form-checkbox rounded text-indigo-600"
+          name="days"
+          value="Monday"
+        />
+        <span class="ml-2 text-gray-700 text-md">Monday</span>
+      </label>
+    </div>
+    <div class="rounded shadow p-2">
+      <label class="inline-flex items-center">
+        <input
+          type="checkbox"
+          class="form-checkbox rounded text-indigo-600"
+          name="days"
+          value="Tuesday"
+        />
+        <span class="ml-2 text-gray-700 text-md">Tuesday</span>
+      </label>
+    </div>
+    <div class="rounded shadow p-2">
+      <label class="inline-flex items-center">
+        <input
+          type="checkbox"
+          class="form-checkbox rounded text-indigo-600"
+          name="days"
+          value="Wednesday"
+        />
+        <span class="ml-2 text-gray-700 text-md">Wednesday</span>
+      </label>
+    </div>
+    <div class="rounded shadow p-2">
+      <label class="inline-flex items-center">
+        <input
+          type="checkbox"
+          class="form-checkbox rounded text-indigo-600"
+          name="days"
+          value="Thursday"
+        />
+        <span class="ml-2 text-gray-700 text-md">Thursday</span>
+      </label>
+    </div>
+    <div class="rounded shadow p-2">
+      <label class="inline-flex items-center">
+        <input
+          type="checkbox"
+          class="form-checkbox rounded text-indigo-600"
+          name="days"
+          value="Friday"
+        />
+        <span class="ml-2 text-gray-700 text-md">Friday</span>
+      </label>
+    </div>
+    <div class="rounded shadow p-2">
+      <label class="inline-flex items-center">
+        <input
+          type="checkbox"
+          class="form-checkbox rounded text-indigo-600"
+          name="days"
+          value="Saturday"
+        />
+        <span class="ml-2 text-gray-700 text-md">Saturday</span>
+      </label>
+    </div>
+    <div class="rounded shadow p-2">
+      <label class="inline-flex items-center">
+        <input
+          type="checkbox"
+          class="form-checkbox rounded text-indigo-600"
+          name="days"
+          value="Sunday"
+        />
+        <span class="ml-2 text-gray-700 text-md">Sunday</span>
+      </label>
+    </div>
```

Comment:
What would it look like if these were horizontal instead of vertical? It would make it so you could see all of the information to enter without scrolling.

Not mandatory, just curious if it would look good and allow less scrolling.